queries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ OPTIONAL] The range here should include till the end of 2019, not just the beggining, something like so: 
```
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```
Please kindly fix that.